docs/diracdoctools/Utilities.py

```diff
@@ -129,3 +130,22 @@ def makeLogger(name):
   logging.basicConfig(level=logging.INFO, format='%(name)25s: %(levelname)8s: %(message)s',
                       stream=sys.stdout)
   return logging.getLogger(name)
+
+
+def registerValidatingExitHandler():
+  """Registers an exit handler which checks for errors after the build completes"""
+  def check():
+    """"""
+    cmd = [
+        "grep", "--color", "-nH", "-e", ":param",
+        "-e", ":return", "-r", "build/html/CodeDocumentation/"
```

Comment:
from your build
> The HTML pages are in _build/html.
> Updating searchtools for Read the Docs search... grep: build/html/CodeDocumentation/: No such file or directory


```suggestion
        "-e", ":return", "-r", "_build/html/CodeDocumentation/"
```

And change the builddir to `_build` in the Makefile

Reply:
Fixed now: https://readthedocs.org/projects/cburrdirac/builds/13202691/

I went with crude argument parsing like the other bits in `conf.py` as it currently uses `build/` when running manually.